bookish.py

```diff
@@ -240,7 +240,19 @@ def generateArgoApplicationSpec(self) -> dict:
                 "NUTSHELL_WORKER_MODEL_PREDICT_TIMEOUT_S")},
             {"name": "model.path", "value": self.env_vars.get(
                 "NUTSHELL_MODEL_PATH")}
-        ]}
+        ]
+
+        resource_cpu = int(os.environ.get("INPUT_RESOURCEMEMORY", "-1"))
+        if resource_cpu > -1:
+            self._send_slack_message(f"Modified Kubernetes CPU to {resource_cpu}", self._thread_ts)
+            parameters.append({"name": "resources.cpu", "value": resource_cpu})
+        resource_memory = int(os.environ.get("INPUT_RESOURCECPU", "-1"))
+        if resource_memory > -1:
+            self._send_slack_message(f"Modified Kubernetes CPU to {resource_memory}", self._thread_ts)
+            parameters.append(
+                {"name": "resources.memory", "value": resource_memory})
```

Comment:
```suggestion
        resource_cpu = int(os.environ.get("INPUT_RESOURCEMEMORY", "-1"))
        if resource_cpu > -1:
            self._send_slack_message(f"Set CPU resource amount to {resource_cpu}", self._thread_ts)
            parameters.append({"name": "resources.cpu", "value": resource_cpu})
        resource_memory = int(os.environ.get("INPUT_RESOURCECPU", "-1"))
        if resource_memory > -1:
            self._send_slack_message(f"Set memory resource amount to {resource_memory}", self._thread_ts)
            parameters.append(
                {"name": "resources.memory", "value": resource_memory})
```
I've spotted a mistake in the message for the memory that was saying "CPU". Also I took the freedom to suggest new messages.